src/connections/ProjectPitch.ts

```diff
@@ -0,0 +1,132 @@
+import { createSelector } from "reselect";
+
+import { EntityIndexFilterKey } from "@/connections/Entity";
+import {
+  adminProjectPitchesIndex,
+  EntityIndexQueryParams,
+  projectPitchesGetUUIDIndex
+} from "@/generated/v3/entityService/entityServiceComponents";
+import { ProjectPitchDto } from "@/generated/v3/entityService/entityServiceSchemas";
+import {
+  adminProjectPitchesIndexFetchFailed,
+  adminProjectPitchesIndexIndexMeta,
+  projectPitchesGetUUIDIndexFetchFailed,
+  projectPitchesGetUUIDIndexIsFetching
+} from "@/generated/v3/entityService/entityServiceSelectors";
+import { getStableQuery } from "@/generated/v3/utils";
+import ApiSlice, { ApiDataStore, PendingErrorState, StoreResourceMap } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionLoader } from "@/utils/connectionShortcuts";
+import { selectorCache } from "@/utils/selectorCache";
+
+const projectPitchesSelector = () => (store: ApiDataStore) =>
+  store["projectPitches"] as StoreResourceMap<ProjectPitchDto>;
+
+type ProjectPitchConnection = {
+  isLoading: boolean;
+  requestFailed: PendingErrorState | null;
+  isSuccess: boolean;
+  projectPitch: ProjectPitchDto | any;
+};
+
+type ProjectPitchConnectionProps = {
+  uuid: string;
+};
+
+const projectPitchConnection: Connection<ProjectPitchConnection, ProjectPitchConnectionProps> = {
+  load: ({ projectPitch }, { uuid }) => {
+    if (!projectPitch) projectPitchesGetUUIDIndex({ pathParams: { uuid } });
```

Comment:
Two things here:
* Please remember to use explicit null checks: `projectPitch == null`
* This is not sufficient. If the fetch fails, the request is complete but the pitch will be null so this will just continuously retry to failing fetch. See other connections for examples. In most cases, extracting the `isLoaded` line below to a separate function and calling it here with the connection state is enough.